src/elements/hx-modal/_shadow.scss

```diff
@@ -13,9 +14,9 @@
 }
 
 #hxModal {
-  @box-shadow: 0px 7px 9px 0 fade(@gray-1000, 30%);
```

Comment:
`fade()` is a LESS function that doesn't exist in SASS, but `rgba()` takes similar arguments to get the same result...

LESS | SASS
----- | -----
`fade(@color, 42%)` | `rgba($color, 0.42)`